test/any/web_catalog/api_extractor-integration.es6.js

```diff
@@ -18,325 +18,330 @@
-        },
-        'valueOf': {
-          'writable': 1,
-        },
-      },
-      '10005': {  // FunctionInterface
-        'prototype': {
-          'writable': 1,
-        },
-      },
-      '10006': {  // FunctionInterface.prototype
-        'meaningfulAPI': {
-          'writable': 1,
-        },
-        'caller': {
-          'writable': 1,
-        },
-        'length': {
-          'writable': 1,
-        },
-      },
-      '10007': {  // ObjectInterface
-        'prototype': {
-          'writable': 1,
-        },
-        'protoProperty': {
-          'writable': 1,
-        },
-      },
-      '10008': {  // ObjectInterface.prototype
-        'meaningfulAPI': {
-          'writable': 1,
-        },
-        'valueOf': {
-          'writable': 1,
-        },
-      },
-      '10009': {  // ObjectLibrary
-        'toString': {
-          'writable': 1,
-        },
-        'valueOf': {
-          'writable': 1,
-        },
-        'InterfaceInstance': {
-          'writable': 1,
-        },
-        'ObejectInstanceA': {
-          'writable': 1,
-        },
-        'functionAPI': {
-          'writable': 1,
-        },
-        'property': {
-          'writable': 1,
-        },
-        'constObjectProperty': {
-          'writable': 0,
-        },
-        'constantProperty': {
-          'writable': 0,
-        },
-      },
-      '10010': {  // nonObjectLibrary
-        'toString': {
-          'writable': 1,
-        },
-        'valueOf': {
-          'writable': 1,
-        },
-        'constructor': {
-          'writable': 1,
-        },
-        'toLocaleString': {
-          'writable': 1,
-        },
-      },
-      '10011': {  // FunctionNonInterface
-        'prototype': {
-          'writable': 1,
-        },
-      },
-      '10012': {  // FunctionNonInterface.prototype
-        'caller': {
-          'writable': 1,
-        },
-        'length': {
-          'writable': 1,
-        },
-        'name': {
-          'writable': 1,
-        },
-      },
-      '10013': {
-        'hiddenAPI': {
-          'writable': 1,
-        },
-      },
-      '10014': {
-        'hiddenAPI': {
-          'writable': 1,
-        },
-      },
-      '10015': {
-        'extraProperty': {
-          'writable': 1,
-        },
-      },
-      '10016': {
-        'meaningfulAPI': {
-          'writable': 1,
-        },
-      },
-      '10017': {
-      },
-    },
-    'protos': {
-      '10005': '10016',
-      '10013': '10017',
-      '10014': '10004',
-    },
-    'root': 10000,
-    'types': {
-      'boolean': 2,
-      'exception': 7,
-      'null': 6,
-      'number': 3,
-      'string': 4,
-      'symbol': 5,
-      'undefined': 1,
-    },
-    'key': 'window',
-  });
-  let extractor = org.chromium.apis.web.ApiExtractor.create({});
+  let objectGraph;
+  let og;
+  let extractor;
+  let apiCatalog;
+  beforeEach(function() {
+    objectGraph = global.ObjectGraph;
+    og = objectGraph.fromJSON({
```

Comment:
Kinda-sorta. Initialization needed to be moved into a `beforeEach()` call. That said, the initialization data/logic has not changed.